City-of-Helsinki/berth-reservations-admin
# VEN-619 | Disable creation of berths if no piers

## Description :sparkles:
- Disable create berth button if no piers
- Prevent form submission on delete for piers and berths
- Add component test
### Closes :no_good_woman:
[VEN-619](https://helsinkisolutionoffice.atlassian.net/browse/VEN-619)
### Automated tests :gear:️
Component render test: `IndividualHarborTableTools.test.tsx`
### Manual testing :construction_worker_man:
Check the [JIRA ticket](https://helsinkisolutionoffice.atlassian.net/browse/VEN-619) please.
